components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/TransferResource.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.identity.api.server.application.management.v1.core;
+
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.MediaType;
+
+/**
+ * TransferResource
```

Comment:
Improve class description

Reply:
Class description is updated to :  
"The TransferResource class represents a resource to be transferred, including its name, type, and byte array data."